### ISSUE TITLE
Problem with importing custom packages -- 'NoneType' object is not callable

### ISSUE THREAD

**Issue by fantamiracle** [Miserlou/Zappa#660]
I have a this package called calcium. In the calcium package, I have a set of flask apis, which imports a library file from calcium package: calcium.lib.

The problem is when I install calcium package in the virtual env, I can see the package can be imported. But then when I use `zappa update` to upload my code, it seems like the package is not installed any more neither on aws lambda, nor locally on the box. It seems like `zappa update` removes my package.

Here is what is happening:

```
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from calcium.lib import OPCA, OPResponse, OPDB
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named calcium.lib
>>> exit()
```
Now we install the package:
```
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ cd ../..
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium$ sudo pip install . --upgrade
The directory '/home/XXXX/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
The directory '/home/XXXX/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
Processing /home/XXXX/YYYY/Calcium
Installing collected packages: calcium
  Found existing installation: calcium 0.1.dev0
    Can't uninstall 'calcium'. No files were found to uninstall.
  Running setup.py install for calcium ... done
Successfully installed calcium-0.1.dev0
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium$ 
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium$ python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from calcium.lib import OPCA, OPResponse, OPDB
>>> exit()
```
And when we update AWS with the new package:
```
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium$ cd -
/home/XXXX/YYYY/Calcium/calcium/flask
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ `
Calling update for environment dev..
Downloading and installing dependencies..
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████| 25/25 [00:01<00:00, 16.36pkg/s]
Packaging project as zip..
Uploading flask-dev-1486764915.zip (5.6MiB)..
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████| 5.88M/5.88M [00:01<00:00, 2.99MB/s]
Updating Lambda function code..
Updating Lambda function configuration..
Uploading flask-dev-template-1486764921.json (1.5KiB)..
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████| 1.56K/1.56K [00:00<00:00, 6.03KB/s]
Deploying API Gateway..
Your updated Zappa deployment is live!: https://fmcc3uhokh.execute-api.us-west-2.amazonaws.com/dev
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ 
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ 
Your updated Zappa deployment is live!: https://fmcc3uhokh.execute-api.us-west-2.amazonaws.com/dev
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ 
```
Yet still the problem is there:
```
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ 
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ curl -svo - https://fmcc3uhokh.execute-api.us-west-2.amazonaws.com/dev/TTTT/1234
*   Trying 54.230.87.60...
* Connected to fmcc3uhokh.execute-api.us-west-2.amazonaws.com (54.230.87.60) port 443 (#0)
* found 173 certificates in /etc/ssl/certs/ca-certificates.crt
* found 697 certificates in /etc/ssl/certs
* ALPN, offering http/1.1
* SSL connection using TLS1.2 / ECDHE_RSA_AES_128_GCM_SHA256
* 	 server certificate verification OK
* 	 server certificate status verification SKIPPED
* 	 common name: *.execute-api.us-west-2.amazonaws.com (matched)
* 	 server certificate expiration date OK
* 	 server certificate activation date OK
* 	 certificate public key: RSA
* 	 certificate version: #3
* 	 subject: C=US,ST=Washington,L=Seattle,O=Amazon.com\, Inc.,CN=*.execute-api.us-west-2.amazonaws.com
* 	 start date: Wed, 08 Jun 2016 00:00:00 GMT
* 	 expire date: Sat, 08 Jul 2017 23:59:59 GMT
* 	 issuer: C=US,O=Symantec Corporation,OU=Symantec Trust Network,CN=Symantec Class 3 Secure Server CA - G4
* 	 compression: NULL
* ALPN, server accepted to use http/1.1
> GET /dev/TTTT/1234 HTTP/1.1
> Host: fmcc3uhokh.execute-api.us-west-2.amazonaws.com
> User-Agent: curl/7.47.0
> Accept: */*
> 
< HTTP/1.1 500 Internal Server Error
< Content-Type: application/json
< Content-Length: 700
< Connection: keep-alive
< Date: Fri, 10 Feb 2017 22:15:34 GMT
< x-amzn-RequestId: 7113f4be-efde-11e6-8cc7-d37ee955e4a2
< X-Amzn-Trace-Id: Root=1-589e3b86-f3aed23be444d867e21e8a04
< X-Cache: Error from cloudfront
< Via: 1.1 0826158c5b3316bf0a83b23783f3dc7a.cloudfront.net (CloudFront)
< X-Amz-Cf-Id: ofB446lfNn66RAVaVISdGJ2lBiEsxUUsSs8iuDhYvIo0FSmdkQb11A==
< 
{
    "message": "An uncaught exception happened while servicing this request. You can investigate this with the `zappa tail` command.", 
    "traceback": [
        "Traceback (most recent call last):\n", 
        "  File \"/var/task/handler.py\", line 447, in handler\n    response = Response.from_app(self.wsgi_app, environ)\n", 
        "  File \"/tmp/pip-build-nzWpku/Werkzeug/werkzeug/wrappers.py\", line 865, in from_app\n", 
        "  File \"/tmp/pip-build-nzWpku/Werkzeug/werkzeug/wrappers.py\", line 57, in _run_wsgi_app\n", 
        "  File \"/tmp/pip-build-nzWpku/Werkzeug/werkzeug/test.py\", line 871, in run_wsgi_app\n", 
        "TypeError: 'NoneType' object is not callable\n"
    ]
* Connection #0 to host fmcc3uhokh.execute-api.us-west-2.amazonaws.com left intact
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ zappa tail --since 1m
Calling tail for environment dev..
[1486764926233] No module named calcium.lib: ImportError
Traceback (most recent call last):
  File "/var/task/handler.py", line 520, in lambda_handler
  return LambdaHandler.lambda_handler(event, context)
  File "/var/task/handler.py", line 230, in lambda_handler
  handler = cls()
  File "/var/task/handler.py", line 124, in __init__
  self.app_module = importlib.import_module(self.settings.APP_MODULE)
  File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
  __import__(name)
  File "/var/task/apis.py", line 7, in <module>
  from calcium.lib import OPCA, OPResponse, OPDB
ImportError: No module named calcium.lib
[1486764934138] [DEBUG] 2017-02-10T22:15:34.138Z 711ef0f6-efde-11e6-b2ad-1d483f05d8d6 Zappa Event: {u'body': None, u'resource': u'/{proxy+}', u'requestContext': {u'resourceId': u'g7weou', u'apiId': u'fmcc3uhokh', u'resourcePath': u'/{proxy+}', u'httpMethod': u'GET', u'requestId': u'7113f4be-efde-11e6-8cc7-d37ee955e4a2', u'accountId': u'777708544714', u'identity': {u'apiKey': None, u'userArn': None, u'cognitoAuthenticationType': None, u'accessKey': None, u'caller': None, u'userAgent': u'curl/7.47.0', u'user': None, u'cognitoIdentityPoolId': None, u'cognitoIdentityId': None, u'cognitoAuthenticationProvider': None, u'sourceIp': u'38.75.3.188', u'accountId': None}, u'stage': u'dev'}, u'queryStringParameters': None, u'httpMethod': u'GET', u'pathParameters': {u'proxy': u'TTTT/1234'}, u'headers': {u'Via': u'1.1 0826158c5b3316bf0a83b23783f3dc7a.cloudfront.net (CloudFront)', u'CloudFront-Is-Desktop-Viewer': u'true', u'CloudFront-Is-SmartTV-Viewer': u'false', u'CloudFront-Forwarded-Proto': u'https', u'X-Forwarded-For': u'38.75.3.188, 205.251.202.48', u'CloudFront-Viewer-Country': u'US', u'Accept': u'*/*', u'User-Agent': u'curl/7.47.0', u'X-Amzn-Trace-Id': u'Root=1-589e3b86-14d27c717e64cac673c39eba', u'Host': u'fmcc3uhokh.execute-api.us-west-2.amazonaws.com', u'X-Forwarded-Proto': u'https', u'X-Amz-Cf-Id': u'u0E_swMDbo0BIcSAsYk3SvIIMti8KcmzcDaPlNgf8h2RZATGc5w52Q==', u'CloudFront-Is-Tablet-Viewer': u'false', u'X-Forwarded-Port': u'443', u'CloudFront-Is-Mobile-Viewer': u'false'}, u'stageVariables': None, u'path': u'/TTTT/1234', u'isBase64Encoded': False}
[1486764934139] 'NoneType' object is not callable
(dev_env) XXXX@XXXX-ZZZZ:~/YYYY/Calcium/calcium/flask$ python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from calcium.lib import OPCA, OPResponse, OPDB
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named calcium.lib
```
Basically it seems like the package is not even installed.

Here is my zappa settings:
```
{
   {
    "dev": {
        "app_function": "apis.application", 
        "s3_bucket": "VVVV",
	"aws_region": "us-west-2",
	"debug": true,
	"keep_warm": false
    }
}
```

Any help will be much appreciated.
